### PR TITLE
Drop openSUSE 15.0 and 15.1

### PR DESCRIPTION
**Pull request by moio**
This drops the unsupported distro, to lighten up mirrors a bit.

Builds on top of https://github.com/uyuni-project/sumaform/pull/896 (is best merged after that one)

### CODE REVIEW

**Comment by juliogonzalez** [comment]
@moio maybe you should drop 15.1 as well? It's dead since January 31st already.

**Review by rjmateus — APPROVED** [review]
LGTM
One question. Should the following repos also be changed to not point to Leap 15.0?
https://github.com/uyuni-project/sumaform/blob/master/salt/repos/default.sls#L28-L35

**Comment by juliogonzalez** [comment]
> LGTM
> One question. Should the following repos also be changed to not point to Leap 15.0?
> https://github.com/uyuni-project/sumaform/blob/master/salt/repos/default.sls#L28-L35

Nope! As for SUSE Manager (where SLE15 clients are always built on top off SLE15SP0), for Uyuni the openSUSE Leap 15.X client tools are built on top of Leap 15.0, so that URL never changes. I should have probable named it "15" instead of "15.0" but didn't remember when I created the structure, and now everyone is using it.

**Comment by moio** [comment]
> @moio maybe you should drop 15.1 as well? It's dead since January 31st already.

Done

File: backend_modules/libvirt/host/user_data.yaml [hunk 1]
```diff
@@ -104,22 +104,15 @@ yum_repos:
...
+%{ if image == "opensuse152o" }
 packages: ["qemu-guest-agent"]
 
 runcmd:
   - zypper removerepo --all
 %{ endif }
 
-%{ if image == "opensuse152o" }
-packages: ["qemu-guest-agent"]
+%{ if image == "opensuse153o" }
```
Comment (juliogonzalez) [inline backend_modules/libvirt/host/user_data.yaml:114]: Not sure if we still use 15.2 somewhere. Maybe this should be an `or`?

**Review by juliogonzalez — APPROVED** [review]
Minor comment. Other than that, LGTM.

File: backend_modules/libvirt/host/user_data.yaml (hunk 1 above)
Comment (moio) [reply backend_modules/libvirt/host/user_data.yaml:114]: Not sure I understand - line 107 covers 15.2 and it gets the same package as before: `qemu-guest-agent`. So no change expected for 15.2.

15.3 needs different packages (`avahi` and `nss-mdns`).

Maybe I am missing something?

Comment (juliogonzalez) [reply backend_modules/libvirt/host/user_data.yaml:114]: Ah, right. I got confused by how the diff is showing the stuff.

LGTM.

**Pull request merged**